core/core_test.go

```diff
@@ -34,13 +34,19 @@ func TestChallenges(t *testing.T) {
 		t.Errorf("New tls-sni-01 challenge is not sane: %v", tlssni01)
 	}
 
+	tlssni02 := TLSSNIChallenge02()
```

Comment:
Whoops, missed this, `TestChallenge` now needs to set the `AllowTLS02Challenges` feature for this test to pass (and should probably verify it rejects it when the feature is not enabled). See https://github.com/letsencrypt/boulder/blob/master/wfe/wfe_test.go#L610 for an example of how we set features during tests.